SDWebImage/SDWebImageWebPCoder.m

```diff
@@ -329,6 +333,28 @@ - (nullable UIImage *)sd_rawWebpImageWithData:(WebPData)webpData {
     return image;
 }
 
+// Create and return the correct colorspace by checking the ICC Profile
+- (nonnull CGColorSpaceRef)sd_colorSpaceWithDemuxer:(nonnull WebPDemuxer *)demuxer CF_RETURNS_RETAINED {
+    // WebP contains ICC Profile should use the desired colorspace, instead of default device colorspace
+    // See: https://developers.google.com/speed/webp/docs/riff_container#color_profile
+    
+    WebPChunkIterator chunk_iter;
+    CGColorSpaceRef colorSpaceRef = NULL;
+    
+    int result = WebPDemuxGetChunk(demuxer, "ICCP", 1, &chunk_iter);
```

Comment:
`chunk_iter` needs to do a free operation refer to the `WebP` framework, this may not cause issue for currently used `WebP` version, but it's a potential memory leaks place. I'll try to fix.

Reply:
Seems I miss the `WebPDemuxReleaseChunkIterator` call after that `CGColorSpace` was created.